src/views/Brokering.vue

```diff
@@ -10,51 +10,55 @@
     <ion-content>
       <main>
         <section>
+          <ion-button expand="block" :disabled="!hasPermission(Actions.APP_JOB_UPDATE)" @click="addBatch()" >{{ $t('Create new brokering job') }}</ion-button>
+
+          <ion-item lines="none">
+            <ion-label>
+              <h1>{{ $t('Scheduled Job') }}</h1>
+            </ion-label>
+          </ion-item>
+
           <ion-card>
-            <ion-card-header>
-              <ion-card-title>{{ $t("Routing") }}</ion-card-title>
-            </ion-card-header>
             <ion-item @click="viewJobConfiguration({ id: 'REJ_ORDR', status: getJobStatus(jobEnums['REJ_ORDR'])})" detail button>
               <ion-label class="ion-text-wrap">{{ $t("Rejected orders") }}</ion-label>
               <ion-label slot="end">{{ getTemporalExpression('REJ_ORDR') }}</ion-label>
             </ion-item>
-            <ion-item-divider>
-              <ion-label class="ion-text-wrap">{{ $t("Batches") }}</ion-label>
-              <ion-button :disabled="!hasPermission(Actions.APP_JOB_UPDATE)" fill="clear" @click="addBatch()" slot="end">
-                {{ $t("Add") }}
-                <ion-icon :icon="addCircleOutline" slot="end" />
-              </ion-button>
-            </ion-item-divider>
+          </ion-card>
+
+          <ion-card :button="isDesktop" v-for="batch in orderBatchJobs" :key="batch?.id" detail v-show="batch?.status === 'SERVICE_PENDING'" @click="hasPermission(Actions.APP_JOB_UPDATE) && viewJobConfiguration({ id: batch.enumId, job: batch })">
```

Comment:
I think we should add a data attribute or class based on selected card. Maybe this can be done using a hidden radio button group. What do you think @ymaheshwari1

Reply:
Can check on displaying selected card using ion-radio-group.